styles.css

```diff
@@ -2,4 +2,41 @@ body {
     background-color: lightseagreen;
 }
 
+.demo{
```

Comment:
This is up for conversation, but I personally like the syntax of adding a space between the classname and the first bracket. But either way we should keep it consistent